Model/Media.php

```diff
@@ -12,7 +12,6 @@
 namespace Sonata\MediaBundle\Model;
 
 use Imagine\Image\Box;
-use Sonata\ClassificationBundle\Model\CategoryInterface;
```

Comment:
Sorry but I won't do the test now, I try to change the excpetion and add the Use but at this moment I can't do anymore.